index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class='navbar navbar-expand-lg navbar-light justify-content-between bg-light'>
+        <a class="nav-link" href="#about">About</a>
+        <a class="nav-link" href="#achievements">Achievements</a>
+        <a class="nav-link" href="#Gallery">Gallery</a>
+    </nav>
+    <div class="container">
+        <h1>Jean-Claude Van Damme</h1>
+        
+    </div>
+    <div >
+        <section id="About" class="container-fluid">
+            <div class="row align-items-center content">
+                <div class="col-md-6">
+                    <aside>
+                        <img class ="img-fluid" src="http://t0.gstatic.com/licensed-image?q=tbn:ANd9GcQWMg1IluKS3uwtpWLYNquHCVOd0YtGbn4K5rWhoSULCry5_iGOzRX8QwShQBlQrrkKbcz0R5kKfjIsH8c" alt="" class ="img-fluid">
+                    </aside>
+                </div>
+                <div class="col-md-6 text-center">
+                    <div class="row justify-content-center">
+                        <div class="col-10 col-lg-8 blurb mb-5 mb-md-0">
+                            <h2>About</h2>
+                            <p>Jean-Claude Camille François Van Varenberg .Born 18 October 1960), known professionally as Jean-Claude Van Damme (French: [van dam], Dutch: [vɑn ˈdɑmə]), is a Belgian actor, martial artist, filmmaker, and fight choreographer. Born and raised in Brussels, Belgium, at the age of ten his father enrolled him in martial arts classes, which led to Van Damme participating in several competitions. With the desire of becoming an actor, he moved to the United States in 1982, where he did odd jobs and worked on several films, until he got his break as the lead in the martial arts film Bloodsport (1988)</p> 
+                            <button class="btn btn-primary">learn More</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="achievements">class="container-fluid">
+            <div class="row align-items-center content">
+                <div class="col-md-6">
+                    <aside>
+                        <img class ="img-fluid" src="https://c8.alamy.com/comp/G6HKTG/jean-claude-van-damme-during-the-15th-world-music-awards-at-the-monte-G6HKTG.jpg" alt="" class ="img-fluid">
```

Comment:
please use a _space-less equal sign_ for the class in "img-fluid" in line 45 here because its the standard way of  assigning values to attributes